### PR TITLE
Improving the RTL feature

### PR DESCRIPTION
**Pull request by RachidTagzen**
This update includes the improving of this features:

- Feeds' titles alignement from right to left & vice versa. (Solving this [1141](https://github.com/martinrotter/rssguard/issues/1141))
- Switch the entire messages Tableview Layout Direction. (Solving this [1141](https://github.com/martinrotter/rssguard/issues/1141))
- Prevent message to be marked read when it's selected.

### CODE REVIEW

**Review by RachidTagzen — COMMENTED** [review]
.

File: src/librssguard/gui/messagesview.cpp [hunk 1]
```diff
@@ -556,8 +556,10 @@ void MessagesView::selectionChanged(const QItemSelection& selected, const QItemS
     // Set this message as read only if current item
     // wasn't changed by "mark selected messages unread" action.
     if (!m_processingRightMouseButton) {
-      m_sourceModel->setMessageRead(mapped_current_index.row(), RootItem::ReadStatus::Read);
-      message.m_isRead = true;
+      if (qApp->settings()->value(GROUP(Messages), SETTING(Messages::MarkMessageReadOnSelectionChange)).toBool()) {
```
Comment (martinrotter) [inline src/librssguard/gui/messagesview.cpp:559]: This is not RTL feature, split into separate pull request.

**Comment by martinrotter** [comment]
You likely messed something up because changes from [a21fedb](https://github.com/martinrotter/rssguard/pull/1355/commits/a21fedb20cab97cc77c612b90bf98d53650a7d59) are not visible in final pull request, you likely reverted those changes by mistake in one of your later commits so really nothing much to review for me.
